test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -88,6 +88,35 @@ def setup
     assert_response :forbidden
   end
 
+  test 'GET show returns the policy' do
+    policy = FactoryBot.create(:policy, account: @provider)
+    get admin_api_registry_policy_path(policy, access_token: @access_token.value)
+    assert_response :success
+    json = JSON.parse(response.body)['policy']
+    assert_equal policy.id, json['id']
```

Comment:
Imo with `dig` is safer to don't raise an error if it does not return `policy` 😄 For example like this instead of these 2 lines:
```suggestion
    assert_equal policy.id, JSON.parse(response.body).dig('policy', 'id')
```
or:
```ruby
  json = JSON.parse(response.body)['policy']
  assert_equal policy.id, json.dig('policy', 'id')
```

Reply:
I prefer the way it is right now. It clearly separates what's the json object of interest and its attributes for the assertion(s).

Reply:
@guicassolato Then you can accomplish that without raising an error anyway 😄 like this:
```suggestion
    json = JSON.parse(response.body)['policy'] || {}
    assert_equal policy.id, json['id']
```